packages/ui-components/src/components/tree-item/tree-item.tsx

```diff
@@ -114,14 +118,18 @@ export class KvTreeItem implements IAnchor {
 							'disabled': this.disabled,
 							'selected': this.selected,
 							'highlighted': this.highlighted,
+							'spotlight': this.spotlighted,
 							'loading': this.loading
 						}}
 					>
 						{!this.loading && (
 							<Fragment>
 								{this.requiresToggleButton && (
 									<div class="expander-arrow" onClick={this.toggleClickThrottler}>
-										<kv-icon name={this.expanded ? EIconName.ArrowDropDown : EIconName.ArrowRight} customClass="pk-grey3 icon-24" />
+										<kv-icon
+											name={this.expandIcon}
+											customClass={classNames('pk-grey3 icon-24', { 'rotate-180': this.expanded, 'rotate-90': !this.expanded })}
```

Comment:
```suggestion
											customClass={{{
											    'pk-grey3': true,
											    'icon-24': true,
											    'rotate-180': this.expanded, 
											    'rotate-90': !this.expanded 
											  })}
```

Reply:
we also have the css-class helper inside the `utils` folder with a few methods that could these needs ;)